ekino/newrelic-bundle/2.0/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Ekino\\NewRelicBundle\\EkinoNewRelicBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "#1": "Get the REST API key of your NewRelic Account on https://docs.newrelic.com/docs/apis/rest-api-v2/requirements/api-keys. Using an API key is optional.",
+        "NEWRELIC_API_KEY": "FIXME"
```

Comment:
post-install ? For indicate the configuration to the user ?

Reply:
Thank you. 
I decided not to since this is optional. 

It is only used if you want to use deployment notifications.